scripts/licenses.ml

```diff
@@ -580,8 +580,8 @@ POSSIBILITY OF SUCH DAMAGE.
 Copyright (c) 2009-2018 <the authors, see individual headers on files>
     |} ^isc
   }
-  | "cstruct.6.0.1" | "cstruct-lwt.6.0.1" | "cstruct-sexp.6.0.1" | "ppx_cstruct.6.0.1" -> {
-    link = "https://github.com/mirage/ocaml-cstruct/blob/28dade8963a9edfddeb8dba782a8eae0341e80d4/LICENSE.md";
+  | "cstruct.6.1.0" | "cstruct-lwt.6.1.0" | "cstruct-sexp.6.1.0" | "ppx_cstruct.6.1.0" -> {

```

Comment:
To keep it working on CircleCI, you could keep both of these sets of versions e.g.
```ocaml
| "cstruct.6.0.1" | "cstruct-lwt.6.0.1" | "cstruct-sexp.6.0.1" | "ppx_cstruct.6.0.1" | "cstruct.6.1.0" | "cstruct-lwt.6.1.0" | "cstruct-sexp.6.1.0" | "ppx_cstruct.6.1.0" -> {
```

Reply:
Yes, good point, I’ll do that!